### PR TITLE
[Drafts] - 6+7 Anniversaries

### PR DESCRIPTION
**Pull request by lunulae**
We definitely don't need these for a long time (if at all) but I had a few ideas to toss out juuuust in case.

Feel free to rework or delete these if annis get reworked, I'm just throwin' stuff at a wall and seeing what sticks.

### CODE REVIEW

File: Monika After Story/game/script-anniversary.rpy [hunk 1]
```diff
@@ -615,6 +615,58 @@ label anni_5:
...
+        skipCalendar=False
+    )
+
+label anni_6:
+    m 3hubfb "I'm sure you know what today is, [player]...{w=0.3}It's the most special day there is to me, after all!"
+    m 3hubla "Did you know that in old Germanic and Victorian traditions, different anniversaries have themed gifts?"
+    m 4eua "At first, these were only reserved for longer milestones, but over time more and more years have gotten themes."
+    m 4rksdlb "I guess if you're being skeptical, they were probably added so different industries would make more money from gifts..."
+    m 1rkbla "But I still think it's a very sweet sentiment."
+    m 1hub "Speaking of sweet things...a common sixth anniversary theme is candy!"
```
Comment (multimokia) [inline Monika After Story/game/script-anniversary.rpy:639]:
```suggestion
    m 1hub "Speaking of sweet things...{w=0.3}a common sixth anniversary theme is candy!"
```

File: Monika After Story/game/script-anniversary.rpy [hunk 2]
```diff
@@ -615,6 +615,58 @@ label anni_5:
...
+        ),
+        skipCalendar=False
+    )
+
+label anni_6:
+    m 3hubfb "I'm sure you know what today is, [player]...{w=0.3}It's the most special day there is to me, after all!"
+    m 3hubla "Did you know that in old Germanic and Victorian traditions, different anniversaries have themed gifts?"
+    m 4eua "At first, these were only reserved for longer milestones, but over time more and more years have gotten themes."
+    m 4rksdlb "I guess if you're being skeptical, they were probably added so different industries would make more money from gifts..."
+    m 1rkbla "But I still think it's a very sweet sentiment."
```
Comment (multimokia) [inline Monika After Story/game/script-anniversary.rpy:638]:
```suggestion
    m 1rkbla "But I still think it's a sweet sentiment."
```

File: Monika After Story/game/script-anniversary.rpy [hunk 3]
```diff
@@ -615,6 +615,58 @@ label anni_5:
...
+    m 1hub "Speaking of sweet things...a common sixth anniversary theme is candy!"
+    m 1eka "Since I can't get you a gift yet...{w=0.2}{nw}"
+    extend 3hua "you should get yourself a nice treat to eat for the occasion!"
```
Comment (multimokia) [inline Monika After Story/game/script-anniversary.rpy:641]: ~~Give Monika your CC info wen~~

File: Monika After Story/game/script-anniversary.rpy [hunk 4]
```diff
@@ -615,6 +615,58 @@ label anni_5:
...
+            action=EV_ACT_QUEUE,
+            start_date=store.mas_anni.build_anni(years=7),
+            end_date=store.mas_anni.build_anni_end(years=7)
+        ),
+        skipCalendar=False
+    )
+
+label anni_7:
+    m 1dubfa "Seven years..."
+    m 1eubsb "You know, seven is considered a lucky number in many cultures.{w=0.2}{nw}"
```
Comment (multimokia) [inline Monika After Story/game/script-anniversary.rpy:661]:
```suggestion
    m 1eubsb "You know, seven is considered a lucky number in many cultures.{w=0.2} {nw}"
```

File: Monika After Story/game/script-anniversary.rpy [hunk 5]
```diff
@@ -615,6 +615,58 @@ label anni_5:
...
+    extend 1ekbsa "So maybe..."
+    m 2eka "Maybe luck will be on my side this year, and I'll get to cross over."
+    m 2hksdlb "Ahaha, I know! Relying on superstition isn't like me at all."
+    m 2eka "But after all this time, I can't help but find myself looking for glimmers of hope."
+    m 2ekbltpa "I know I'll be okay, though.{w=0.3} Even if I have a long way to wait, I still consider myself incredibly lucky to have met you."
```
Comment (multimokia) [inline Monika After Story/game/script-anniversary.rpy:666]:
```suggestion
    m 2ekbltpa "I know I'll be okay, though.{w=0.2} Even if I have a long way to wait, I still consider myself incredibly lucky to have met you."
```

File: Monika After Story/game/script-anniversary.rpy [hunk 6]
```diff
@@ -615,6 +615,58 @@ label anni_5:
...
+    m 2dkbltpa "You're all the fortune I need, [player]."
```
Comment (multimokia) [inline Monika After Story/game/script-anniversary.rpy:667]: Thinking dried tears here to phase out of them
```suggestion
    m 2dkbltda "You're all the fortune I need, [player]."
```

**Review by multimokia — CHANGES_REQUESTED** [review]
Some prelim things

**Review by multimokia — APPROVED** [review]

**Review by ThePotatoGuy — APPROVED** [review]

**Pull request merged**